Widgets/Assets.xcassets/WidgetSearchFieldBackgroundColor.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0.918",
+          "green" : "0.918",
+          "red" : "0.918"
+        }
+      },
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0.200",
+          "green" : "0.200",
+          "red" : "0.200"
+        }
+      },
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
1. Why background colors of the widgets differ? If I put two next to each other these look weird.
2. Dark theme background color seems really dark - I'd consider matching our home screen background color ar at least making it a bit lighter.

Reply:
1) Hmm, they don't.  Each widget's background is set with this line:

`Rectangle().fill(Color.widgetBackground)`

There's a "WidgetBackground" color defined in the asset catalog but that doesn't appear to do anything, but even so I didn't want to delete it in case iOS was expecting it.  But `Color.widgetBackground` resolves to WidgetBackgroundColor.

I've just checked with Pixelmator and the look identical to me.  Can you share a screenshot of what you mean?

2) We tried that originally, but Robert didn't like it compared to the Apple widgets so we decided to darken it - it's been Design and Product Reviewed now so I'm not really wanting to change it at this point.

Reply:
Weird, I checked colors and there definitely has been a difference RGB 16/16/16 vs 17/17/17 but after restarting it looks ok... 🤷‍♂️Seems to be fine now 😅